store/src/java/com/zimbra/cs/mailclient/smtp/SmtpTransport.java

```diff
@@ -241,7 +242,11 @@ public void sendMessage(Message msg, Address[] rcpts) throws MessagingException
                 connection.sendMessage(rcpts, (MimeMessage) msg);
             }
         } catch (MessagingException e) {
-            ZimbraLog.smtp.warn("Failed to send message", e);
+            // After receipt validation , messaging exception is always thrown while scheduling msg
```

Comment:
// after receipt validation , messaging exception is always thrown while scheduling msg

We follow a convention that comments to start with lowercase letters

Reply:
Made required changes